clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -36,8 +36,13 @@ include "mlir/IR/SymbolInterfaces.td"
 // CIR Ops
 //===----------------------------------------------------------------------===//
 
+class LoweringInfo {
+  bit SimpleLowering = 0;
```

Comment:
Is this `SimpleLowering` as opposed to other types of lowering? My suggestion is to tie it up for now with the only implemented thing, which is builtins: `bit builtinLowering = 0`;